artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/ssl/DefaultSSLContextFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.core.remoting.impl.ssl;
+
+import org.apache.activemq.artemis.spi.core.remoting.ssl.SSLContextFactory;
+import java.util.Map;
+import javax.net.ssl.SSLContext;
+import org.apache.activemq.artemis.core.remoting.impl.netty.TransportConstants;
+import static org.apache.activemq.artemis.spi.core.remoting.ssl.SSLContextFactory.LOGGER;
+import org.apache.activemq.artemis.utils.ConfigurationHelper;
+
+/**
+ * Simple SSLContextFactory for use in NettyConnector and NettyAcceptor.
+ */
+public class DefaultSSLContextFactory implements SSLContextFactory {
+
+   @Override
+   public SSLContext getSSLContext(Map<String, Object> configuration,
+           String keystoreProvider, String keystorePath, String keystorePassword,
+           String truststoreProvider, String truststorePath, String truststorePassword,
+           String crlPath, String trustManagerFactoryPlugin, boolean trustAll) throws Exception {
+      return createSSLContext(configuration,
+              keystoreProvider, keystorePath, keystorePassword,
+              truststoreProvider, truststorePath, truststorePassword,
+              crlPath, trustManagerFactoryPlugin, trustAll);
+   }
+
+   protected SSLContext createSSLContext(Map<String, Object> configuration,
+           String keystoreProvider, String keystorePath, String keystorePassword,
+           String truststoreProvider, String truststorePath, String truststorePassword,
+           String crlPath, String trustManagerFactoryPlugin, boolean trustAll) throws Exception {
+      if (LOGGER.isDebugEnabled()) {
```

Comment:
You should probably squash both commits.

like this commit here wouldn't compile as you didn't declare LOGGER, and the next one you remove this LOGGER.


As a matter of fact, I would call it log just like everywhere else, but that would be too much OCD on my side.